pavelib/assets.py

```diff
@@ -240,17 +292,16 @@ def compile_sass(options):
         output_style = 'compressed'
 
     timing_info = []
-    system_sass_directories = applicable_sass_directories(systems)
-    all_sass_directories = applicable_sass_directories()
+    system_sass_directories = sass_source_directories(systems)
+    all_sass_directories = sass_lookup_directories()
```

Comment:
Nit: Can we change the variable name from `all_sass_directories` to something like `system_sass_lookup_directories`?